style.css

```diff
@@ -64,3 +74,46 @@ h1 {
     font-size: 20px;
 }
 
+.blog {
+    background-color: #4c5c96;
+    padding: 60px 0;
+}
+
+.third-header {
+    font-size: 24px;
+    text-transform: uppercase;
+    margin-bottom: 24px;
+    text-align: left;
+}
+
+.artile-text {
+    color: #ccc;
+    font-size: 20px;
+    display: block;
+    margin-bottom: 24px;
+}
+
+.article-link {
+    color: #fff;
+    font-size: 20px;
+    text-transform: uppercase;
+    text-decoration: none;
+}
+
+.blog-two {
+    background-color: #0d2e41;
+}
+
+.blog-third {
+    background-color: #1d7aaf;
+}
+
+/* Footer */
```

Comment:
Make sure to use a same naming convention in the comments too.

Like : 
```
/* Header */
/* Footer */
```

Or: 
```
/* Header Styles */
/* Footer Styles*/
```